src/pool/srv_pool.c

```diff
@@ -1310,6 +1344,26 @@ read_db_for_stepping_up(struct pool_svc *svc, struct pool_buf **map_buf,
 	return rc;
 }
 
+int
+ds_pool_svc_rf_to_nreplicas(int svc_rf)
+{
+	D_ASSERTF(daos_svc_rf_is_valid(svc_rf), "%d out of range\n", svc_rf);
+	return svc_rf * 2 + 1;
+}
+
+int
+ds_pool_svc_rf_from_nreplicas(int nreplicas)
+{
+	int svc_rf;
+
+	D_ASSERTF(nreplicas > 0, "%d out of range\n", nreplicas);
+	if (nreplicas % 2 == 0)
+		svc_rf = (nreplicas - 1) / 2;
+	else
+		svc_rf = nreplicas / 2;
```

Comment:
[Note] To address @wangshilong's comment on upgrading a 13-PS-replicas 2.2 pool, I plan to add

	if (svc_rf > DAOS_PROP_PO_SVC_REDUN_FAC_MAX)
		svc_rf = DAOS_PROP_PO_SVC_REDUN_FAC_MAX;

here.